runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc

```diff
@@ -54,22 +55,31 @@ void ElementwiseActivationLayer::configure(const IPortableTensor *input, IPortab
     case ElementwiseActivationType::kReLU:
       if (input->data_type() == OperandType::FLOAT32)
       {
-        if (alpha == std::numeric_limits<float>::infinity() && beta == 0.f)
+        if ((alpha == std::numeric_limits<float>::infinity() || alpha == 6.0f) && beta == 0.f)
         {
           cpu::ops::ElementwiseActivationLayer::configure(
             input, output, alpha, beta, cpu::ops::ElementwiseActivationType::kReLU);
 
-          _backward_kernel = [](const IPortableTensor *output, const IPortableTensor *incoming,
-                                IPortableTensor *outgoing) {
-            nnfw::cker::train::ReLUGrad(getShape(output), getBuffer<float>(output),
-                                        getShape(incoming), getBuffer<float>(incoming),
-                                        getShape(outgoing), getBuffer<float>(outgoing));
+          auto relu_cker = [&alpha]() {
+            if (alpha == std::numeric_limits<float>::infinity())
+              return nnfw::cker::train::ReLUGrad;
+            else if (alpha == 6.0f)
+              return nnfw::cker::train::ReLU6Grad;
+            else
+              throw std::runtime_error{"no supported relu kernel"};
+          }();
+
+          _backward_kernel = [relu_cker](const IPortableTensor *output,
+                                         const IPortableTensor *incoming,
+                                         IPortableTensor *outgoing) {
+            relu_cker(getShape(output), getBuffer<float>(output), getShape(incoming),
+                      getBuffer<float>(incoming), getShape(outgoing), getBuffer<float>(outgoing));
           };
         }
         else
         {
           throw std::runtime_error("train ElementwiseActivationLayer : This layer does not "
-                                   "suppport other ReLU except for ReLU(0-inf)");
+                                   "suppport other ReLU except for ReLU(0-inf) and ReLU6(0-6)");
         }
```

Comment:
### NOTE
# WIP

After This PR, the training feature supports `ReLU6`. (not fused one) 

I also checked loss value isn't different with tensorflow. 



* TensorFlow
```
Epoch 1/5
50/50 [==============================] - 0s 617us/step - loss: 0.0794 - mean_squared_error: 0.0794
Epoch 2/5
50/50 [==============================] - 0s 542us/step - loss: 0.0586 - mean_squared_error: 0.0586
Epoch 3/5
50/50 [==============================] - 0s 533us/step - loss: 0.0505 - mean_squared_error: 0.0505
Epoch 4/5
50/50 [==============================] - 0s 542us/step - loss: 0.0461 - mean_squared_error: 0.0461
Epoch 5/5
50/50 [==============================] - 0s 513us/step - loss: 0.0431 - mean_squared_error: 0.0431
```

* onert_train
```
- learning_rate   = 0.001
- batch_size      = 20
- loss_info       = {loss = mean squared error, reduction = sum over batch size}
- optimizer       = adam
========================
Epoch 1/5 - time: 3.351ms/step - loss: [0] 0.0794
Epoch 2/5 - time: 3.375ms/step - loss: [0] 0.0586
Epoch 3/5 - time: 3.409ms/step - loss: [0] 0.0505
Epoch 4/5 - time: 3.655ms/step - loss: [0] 0.0461
Epoch 5/5 - time: 3.915ms/step - loss: [0] 0.0431
===================================
```